### ISSUE TITLE
Panel - Add panel navigation (back button) to panel

### ISSUE THREAD

**Issue by MrDinsdale** [sky-uk/toolkit#313]
## Expected Behaviour
The Design library now features back button placement for panels, we should add styling for this to improve consistency

## Current Behaviour
Each implementation is custom on our panels so often placement and size is inconsistent.

**Comment by joebell93** [comment]
Wondering if this is better being a generic link class for use outside of panels?

Very much like `.c-link-external`, we could have `.c-link-back`

**Comment by MrDinsdale** [comment]
Yeah, the only thing I'd point out with that is the fact we need to consider the positioning of the link as well as it's appearance.

**Comment by joebell93** [comment]
We can do a cheeky BEM mix 🔥

**Comment by joebell93** [comment]
Replicating the following designs from Sketch:

![screen shot 2017-11-30 at 10 25 25](https://user-images.githubusercontent.com/7349341/33426189-e3fd8c6a-d5b8-11e7-9256-03d9a0568b60.png)
![screen shot 2017-11-30 at 10 25 44](https://user-images.githubusercontent.com/7349341/33426194-e6b620c0-d5b8-11e7-9ad7-07d54a573a00.png)

---

@mikejgregory just checking that it's intentional for the back link to be blue like a standard link when the close button is text-coloured?

**Comment by mikejgregory** [comment]
@joebell93 Currently yes in this instance the back button is blue, however we should be building these components so the colour as well as the icon plus it's placement (left or right) should be flexible.

**Comment by joebell93** [comment]
It's a little complicated to explain, but the answer to that is yes and no due to us using `png`s. I'll have a chat later with you

**Comment by joebell93** [comment]
@sky-uk/toolkit-owners I'm thinking we split this into a short-term and long-term option:

---

Short-term

```
<article class="c-panel is-open">
  <div class="c-panel__content">
    <a class="c-panel__back c-link-back" href="#1">Back</a>
    <a class="c-panel__toggle" href="#1">Close</a>
    <div class="o-container">
      ...
    </div>
  </div>
</article>
```

---

Long-term

The extra `c-panel__nav` would remove any need to absolutely position navigation, however, it would drastically change the padding of the panel at the top edge (one for 3.0)

```
<article class="c-panel is-open">
  <div class="c-panel__content">
    <div class="c-panel__nav">
      <a class="c-panel__back c-link-back" href="#1">Back</a>
      <a class="c-panel__toggle" href="#1">Close</a>
    </div>
    <div class="o-container">
      ...
    </div>
  </div>
</article>
```

**Comment by joebell93** [comment]
@mikejgregory has also confirmed to me that the panel navigation *should* be sitting within the container; this would be a breaking change to as it would require restructuring of our Panel padding
